pages/archive/content.py

```diff
@@ -35,38 +41,76 @@ class Content(Page):
         'doctype',
         'buyLink',
         'submitter',
-        'baked',
+        'collated',
         'parentAuthors'
     ]
+    # Whitelisted fields inside the "tree" field
+    _stable_tree_fields = ['contents', 'title']
 
     @property
     def json_pre(self):
+        """Returns the element that wraps the full archive json."""
         return self.find_element(*self._json_locator)
 
     @property
     def json_text(self):
+        """Returns the full archive json as a string."""
         return self.json_pre.text
 
     @property
     @lru_cache(maxsize=None)
     def dict(self):
+        """Returns the archive json as a dict."""
         import json
         return json.loads(self.json_text)
 
     @property
     def id(self):
+        """Returns the value of the "id" field."""
         return self.dict['id']
 
     @property
     def title(self):
+        """Returns the value of the "title" field."""
         return self.dict['title']
 
+    @property
+    def has_tree(self):
+        """Returns whether or not the "tree" field is present."""
+        return 'tree' in self.dict
+
+    @property
+    def tree(self):
+        """Returns the archive tree json as a dict."""
+        return self.dict['tree']
+
+    def stable_tree(self, tree):
+        """Returns the archive tree json as a dict with fields that change
```

Comment:
Your docstrings need to be changed to match the style guide.

Essentially, if you can write it on one line that's great but if it is two lines then it should be a very brief description followed by a new line and then a deeper description.

```python
# Good
def login():
"""Logs in.

Clicks the login link and then waits for the home page to load.

"""
```
```python
# Bad
def login():
"""Logs in.
Clicks the login link and then waits for the home page to load."""
```